src/main/java/com/yugabyte/sample/apps/MultiTableMultiIndexInserts.java

```diff
@@ -0,0 +1,449 @@
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setInt(1, getNextInt(threadIdx));
+      preparedStatement.setString(2, getRandomString(threadIdx));
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA3Statement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "a3");
+    int count = rowsCount.get("a3");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setInt(2, getNextInt(threadIdx));
+      preparedStatement.setString(3, "");
+      preparedStatement.setString(4, "0101000020E61000002A05DD5E525953C0F74773BF9EA54640");
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getA3AStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "a3a");
+    int count = rowsCount.get("a3a");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setObject(2, getNextUUID(threadIdx));
+      preparedStatement.setInt(3, getNextInt(threadIdx));
+      preparedStatement.setInt(4, getNextInt(threadIdx));
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getMStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "m");
+    int count = rowsCount.get("m");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setObject(1, getNextUUID(threadIdx));
+      preparedStatement.setString(2, "abcdefghij");
+      preparedStatement.setString(3, "abcdefghij");
+      preparedStatement.setString(4, "abcdefghij");
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+
+  private PreparedStatement getMVStatement(int threadIdx) throws Exception {
+    PreparedStatement preparedStatement = getPreparedStatement(threadIdx, "mv");
+    int count = rowsCount.get("mv");
+    for (int i = 0; i < count; ++i) {
+      preparedStatement.setInt(1, getNextInt(threadIdx));
+      preparedStatement.setString(2, "abcdefghij");
+      preparedStatement.addBatch();
+    }
+
+    return preparedStatement;
+  }
+  public long doWrite(int threadIdx) {
+    try {
+      PreparedStatement preparedStatement = getA1Statement(threadIdx);
+      getA1AStatement(threadIdx).executeBatch();
+      getA2Statement(threadIdx).executeBatch();
+      getDStatement(threadIdx).executeBatch();
+      getA3Statement(threadIdx).executeBatch();
+      getA3AStatement(threadIdx).executeBatch();
+      getMStatement(threadIdx).executeBatch();
+      getMVStatement(threadIdx).executeBatch();
+      return 1;
+    } catch (Exception e) {
+      LOG.error("...", e);
```

Comment:
By `"..."`, I actually meant the message that you had in the log statement. I did not mean the string `"..."` literally.